src/main/webapp/app/exam/manage/exams/exam-update.component.html

```diff
@@ -73,8 +73,9 @@ <h5 class="pb-1" jhiTranslate="artemisApp.examManagement.sections.conduction"></
                         labelName="{{ 'artemisApp.examManagement.visibleDate' | artemisTranslate }}"
                         labelTooltip="{{ 'artemisApp.examManagement.visibleDateTooltip' | artemisTranslate }}"
                         [(ngModel)]="exam.visibleDate"
-                        [error]="!isValidVisibleDate"
+                        [error]="!isVisibleDate"
```

Comment:
The other two date fields you've changed have `[error]="!isStartDate || !isValidStartDate"` and `"!isEndDate || !isValidEndDate"`.

I think you need to add it here as well; otherwise, it'll be possible to click "Save" when there's no visibleDate

Reply:
Actually, we don't need it because we use `isValidStartDate` and `isValidEndDate` functions to check if they are in the correct order(For example, the start date can not be later than the end date). For the visible date, the checks are done in `isValidStartDate` function.

Reply:
Tested on TS3.

1. Navigate to Exam management > Create new exam
2. Enter a title for the exam
3. Observe the save button at the bottom of the exam; it should be disabled.
4. Pick valid dates for start and end of working time. Enter "foobar" into the "Visible from" field.
5. The save button is now available despite the invalid "Visible from" date


The check you mention in `isValidStartDate` returns `true` because `isValidVisibleDate` is false in this case, but the save button isn't locked because `[error]="!isVisibleDate"` does not take this into account.

Reply:
Fixed.